tmt/steps/__init__.py

```diff
@@ -2444,7 +2444,7 @@ def _login(
         for guest in self.parent.plan.provision.ready_guests:
             # Attempt to push the workdir to the guest
             try:
-                guest.push()
+                guest.push(self.parent.plan.worktree)
```

Comment:
I believe this is already the default if we pass `None` isn't it? No complains on making it more explicit, but could we make it `source=..., destination=None` or such?